src/components/Exchange/ConfirmationBox.js

```diff
@@ -858,18 +846,26 @@ export default function ConfirmationBox(props) {
     currentExecutionFee,
     currentExecutionFeeUsd,
   ]);
+  const submitButtonRef = useRef(null);
+  useKey("Enter", () => {
+    if (submitButtonRef.current) {
```

Comment:
why do we need to check submit button exists? and is there a case when it does not exist?

Reply:
Remove the check. Thanks